docs/development.md

```diff
@@ -24,6 +25,12 @@ cd $GOPATH/src/github.com/redhat-developer/odo
 git remote add upstream 'https://github.com/redhat-developer/odo'
 ```
 
+While cloning Odo, windows terminal like PowerShell or Command Prompt might throw error of `Filename too long`. To avoid such type of issue set the git config `core.longpaths` to `true`.
```

Comment:
`Windows`

Reply:
Windows terminal's such as PowerShell or `CMD` may throw an error of `Filename too long`.

Reply:
To avoid such an error, set your Git config as so:

```sh
git config --global core.longpaths true
```

Reply:
Cool!!!, Will update the pr
I was expecting review from native English speaker @cdrage :)
Thanks @cdrage

Reply:
Done

Reply:
I think it should be 
```
Windows' terminals such as ...
```
or even 
```
terminals of Windows such as...
```
@cdrage  WDYT?

Reply:
@mik-dass Ahh...I think it is right in both the ways as you suggested above.